test/expr/unary/keypath/keypath.swift

```diff
@@ -401,8 +401,8 @@ func testKeyPathSubscriptMetatype(readonly: Z.Type, writable: inout Z.Type,
   sink = readonly[keyPath: rkp]
   sink = writable[keyPath: rkp]
 
-  readonly[keyPath: kp] = sink // expected-error{{cannot assign through subscript: 'kp' is a read-only key path}}
-  writable[keyPath: kp] = sink // expected-error{{cannot assign through subscript: 'kp' is a read-only key path}}
+  readonly[keyPath: kp] = sink // expected-error{{cannot assign through subscript: 'kp' is read-only}}
```

Comment:
I like the old wording better here too because otherwise it's unclear what is `kp`. When I said - let's drop "key path" at the end - I meant only in the situation when we'd already have that in the beginning of the message, one of the messages was something like - `key path is read-only key path` so `key path` bit doesn't really make much sense to repeat.

Reply:
Ahh, I misunderstood your comment. I’ll switch that around